src/main/scala/com/github/unchama/seichiassist/subsystems/bookedachivement/bukkit/command/AchievementCommand.scala

```diff
@@ -114,9 +113,23 @@ object AchievementCommand {
                   case AchievementOperation.GIVE => playerData.tryForcefullyUnlockAchievement(achievementNumber)
                   case AchievementOperation.DEPRIVE => playerData.forcefullyDepriveAchievement(achievementNumber)
                 }
+
               case None =>
-                MessageEffect(s"$playerName は現在サーバーにログインしていません。")
-              // TODO 実績付与予約システムに書き込むようにする
+                effectEnvironment.runEffectAsync("実績を予約する",
+                  service.writeAchivementId(playerName, achievementNumber, operation).start
+                )
+
+                MessageEffect(
+                  List(
+                    s"$playerName の No.$achievementNumber の実績を${
+                      operation match {
+                        case AchievementOperation.GIVE => "付与"
+                        case AchievementOperation.DEPRIVE => "剥奪"
+                      }
+                    }します。",
+                    s"$playerName は現在サーバーにログインしていません。\n予約システムに書き込みました。"
+                  )
+                )
```

Comment:
```suggestion
                SequentialEffect(
                  Kleisli.liftF(
                    service.writeAchivementId(playerName, achievementNumber, operation).start
                  ),
                  MessageEffect(
                    List(
                      s"$playerName の No.$achievementNumber の実績を${
                        operation match {
                          case AchievementOperation.GIVE => "付与"
                          case AchievementOperation.DEPRIVE => "剥奪"
                        }
                      }します。",
                      s"$playerName は現在サーバーにログインしていません。\n予約システムに書き込みました。"
                    )
                  )
                )
```
とすると `effectEnvironment` への言及が消えそう